get-started/index.md

```diff
@@ -54,115 +54,83 @@ redirect_from:
-        Status: Downloaded newer image for hello-world:latest
+![Tutorial container running in Docker Dashboard](images/tutorial-in-dashboard.png)
 
-        Hello from Docker!
-        This message shows that your installation appears to be working correctly.
-        ...
-    ```
 
-2.  Run `docker image ls` to list the `hello-world` image that you downloaded to your machine.
+## What is a container?
 
-3.  List the `hello-world` container (spawned by the image) which exits after displaying its message. If it is still running, you do not need the `--all` option:
+Now that you've run a container, what _is_ a container? Simply put, a container is
+simply another process on your machine that has been isolated from all other processes
+on the host machine. That isolation leverages [kernel namespaces and cgroups](https://medium.com/@saschagrunert/demystifying-containers-part-i-kernel-space-2c53d6979504), features that have been 
+in Linux for a long time. Docker has worked to make these capabilities approachable and easy to use.
 
-    ```shell
-        $ docker ps --all
+!!! info "Creating Containers from Scratch"
+    If you'd like to see how containers are built from scratch, Liz Rice from Aqua Security
+    has a fantastic talk in which she creates a container from scratch in Go. While she makes
+    a simple container, this talk doesn't go into networking, using images for the filesystem, 
+    and more. But, it gives a _fantastic_ deep dive into how things are working.
 
-        CONTAINER ID     IMAGE           COMMAND      CREATED            STATUS
-        54f4984ed6a8     hello-world     "/hello"     20 seconds ago     Exited (0) 19 seconds ago
-    ```
+    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/8fi7uSYlOdc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-## Conclusion
+## What is a container image?
 
-At this point, you've installed Docker Desktop on your development machine, and ran a quick test to ensure you are set up to build and run your first containerized application.
+When running a container, it uses an isolated filesystem. This custom filesystem is provided 
+by a **container image**. Since the image contains the container's filesystem, it must contain everything 
+needed to run an application - all dependencies, configuration, scripts, binaries, etc. The 
+image also contains other configuration for the container, such as environment variables,
+a default command to run, and other metadata.
 
-[On to Part 2 >>](part2.md){: class="button outline-btn" style="margin-bottom: 30px; margin-right: 100%"}
+We'll dive deeper into images later on, covering topics such as layering, best practices, and more.
 
-For information on how to build and run your first containerized application using Node.js, go to [Build your Node.js image](/nodejs/build-images.md).
+!!! info
```

Comment:
Wondering if this is an admonition? We can change this to a note to be consistent with the existing format